expression/builtin_arithmetic.go

```diff
@@ -523,7 +524,7 @@ func (s *builtinArithmeticMultiplyDecimalSig) evalDecimal(row types.Row) (*types
 	}
 	c := &types.MyDecimal{}
 	err = types.DecimalMul(a, b, c)
-	if err != nil {
+	if err != nil && !terror.ErrorEqual(err, types.ErrTruncated) {
```

Comment:
```
  conventions:
    decimal_smth() <= 1     -- result is usable, but precision loss is possible
```
In [MySQL]( https://github.com/mysql/mysql-server/blob/220d2eb9623594f3bf3c81cf4f8af45c210c233b/include/decimal.h#L116) multiple will ignore mul DataTruncated error without any error or warning.....

Reply:
Do we have some test to cover this behavior?